packages/utils.js

```diff
@@ -117,9 +109,21 @@ export function throttle (callback, ms) {
   }
 }
 
+/**
+ * toggleAttribute (Ponyfill for IE and Edge, fixes #299)
+ * @param {Element} el  The new throttled function
```

Comment:
`The new throttled function` kan fjernes

Reply:
hehe, takk, typo :)